.github/workflows/audit.yml

```diff
@@ -49,3 +49,7 @@ jobs:
       with:
         name: integration-test-results-${{ matrix.audit }}-${{ matrix.java }}
         path: build/audit/integration-test-results
+
+concurrency:
+  group: audit-${{ github.event.pull_request.number || github.sha }}
```

Comment:
Thanks for this. I used it to do same thing in `python-gitlab`.

One thing I did different was:

```
group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
```

Just a thought if you wanted to do that.